Client/Frontend/Theme/ThemeManager.swift

```diff
@@ -64,18 +82,95 @@ final class DefaultThemeManager: ThemeManager {
         return currentTheme.type.getInterfaceStyle()
     }
 
+    func changeCurrentTheme(_ newTheme: ThemeType) {
+        guard currentTheme.type != newTheme else { return }
+        // TODO: Check for nightmode
```

Comment:
Nit, if we have a task for it I would suggest putting it in the TODO
```suggestion
        // TODO: Check for nightmode
```